scripts/test/jest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+jest
```

Comment:
I just went for consistency for the sake of getting this task done in a reasonable time (by using what I know). We can definitely revisit this as I'm in no way married to scripty. I've used it on a couple of large projects, and it makes structuring your tasks a bit easier, but if we don't see the need then I say we find a better (simpler) solution.